cloudinary-core/src/main/java/com/cloudinary/Util.java

```diff
@@ -31,6 +31,7 @@
         params.put("allowed_formats", StringUtils.join(ObjectUtils.asArray(options.get("allowed_formats")), ","));
         params.put("moderation", options.get("moderation"));
         params.put("access_mode", (String) options.get("access_mode"));
+        params.put("quality_override", options.get("quality_override"));
```

Comment:
This is not needed here - quality_override is a 'write param' (it's used both in upload/explicit and in update) so it should be part of 'processWriteParams'. You did put it there, so it's not needed here. 

It's needed in the 'else' branch of the signature instead (line 54 and on). 
As it's written right now, if someone calls update with this param, but did his own signing, this param will get lost.